README.md

```diff
@@ -3,11 +3,11 @@ The Poseidon/Firmament scheduler incubation project is to bring integration of F
 At a very high level, Poseidon/Firmament scheduler augments the 
 current Kubernetes scheduling capabilities by incorporating a new 
 novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. 
-Firmament models workloads and clusters as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
+Firmament models workloads on a cluster as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
 
-Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling environment that constantly keeps refining the workloads placements dynamically.
+Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling environment that keeps on refining the dynamic placements of the workload.
```

Comment:
"the new scheduler enables a globally optimal scheduling environment" -- this is misleading, since Firmament is only optimal for a given policy. This should either be clear, or claims of optimality should be dropped.